hardware/neopixel/neopixel.js

```diff
@@ -35,6 +35,8 @@ module.exports = function(RED) {
         this.fgnd = n.fgnd || "128,128,128";
         this.mode = n.mode || "pcent";
         this.rgb = n.rgb || "rgb";
+        this.gamma = n.gamma;
```

Comment:
Doesn't this need to be defaulted also ?

Reply:
agreed

Reply:
done